src/Raven.Server/settings.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-    "ServerUrl": "http://127.0.0.1:0",
-    "Setup.Mode": "Initial"
+    "ServerUrl": "http://127.0.0.1:8080",
+    "Setup.Mode": "None"
```

Comment:
The logic here is that this is what we use for development.
We need to update the `settings.json` we use for _deployments_, we'll get that with the RC2 deployment work